packages/dashboard/src/components/Wizard/examples/SetupWizard.Util.ts

```diff
@@ -0,0 +1,222 @@
+import { ISubwayNavStep, SubwayNavStepState } from '../../SubwayNav/SubwayNav.types';
+import { IWizardStepProps } from '../Wizard.types';
+
+export interface ISetupWizardState {
+  currStep: IWizardStepProps;
+  currSubStep: IWizardStepProps | undefined;
+  steps: IWizardStepProps[];
+}
+
+/**
+ * generate Random id
+ */
+export function generateRandomId(): string {
+  return (
+    Math.random()
+      .toString(36)
+      .substring(2) + new Date().getTime().toString(36)
+  );
+}
+
+// Go to a given parent step -> substep
+export function goToStep(
+  currentState: ISetupWizardState,
+  parentStepToGo: ISubwayNavStep,
+  subStepToGo: ISubwayNavStep | undefined
+): ISetupWizardState | undefined {
```

Comment:
similar to a previous comment, is it a valid scenario to be "unable to find the step"?

Reply:
This is just a test code for the example.  I still need to work on this more.  Will continue to look at refactoring.